src/libraries/System.Net.Sockets/tests/FunctionalTests/SocketDuplicationTests.cs

```diff
@@ -15,7 +15,7 @@
 
 namespace System.Net.Sockets.Tests
 {
-    // Test cases for DuplicateAndClose, Socket(socketInformation), Socket.UseOnlyOverlappedIO,
+    // Test cases for DuplicateAndClose, Socket(socketInformation), Socket(socketInformation),
```

Comment:
Also revert this comment, tests still incude Socket.UseOnlyOverlappedIO

Reply:
👍